libs/ui/search/src/lib/organisation-preview/organisation-preview.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="">
+  <div
+    class="overflow-hidden bg-gray-100 rounded-lg w-full border border-gray-300 h-36"
+  >
+    <gn-ui-record-thumbnail
+      class="relative h-full w-full object-cover object-left-top"
+      [thumbnailUrl]="logo"
+    ></gn-ui-record-thumbnail>
+  </div>
+  <div class="px-3 capitalize">
+    <span
+      [title]="title"
+      class="mb-3 mt-5 font-title text-21 text-title line-clamp-2 sm:line-clamp-1 sm:mt-2"
+    >
+      {{ title }}</span
+    >
+    <p
+      class="abstract mt-4 mb-5 h-36 sm:mb-2 sm:h-[4.5rem] line-clamp-6 sm:line-clamp-3 sm:mt-0"
+    >
+      {{ description }}
+    </p>
+    <!--
+    <button
+      translate
+      class="flex items-center justify-center capitalize bg-[#c2cfe3] text-white p-1 rounded text-sm my-2 p-0 pr-1"
+    >
+      <mat-icon class="flex text-base">add </mat-icon>
+      <span>suivre</span>
+    </button>
+-->
+    <div class="text-primary opacity-25 flex leading-6">
+      <mat-icon class="text-primary opacity-25 mr-1">folder_open </mat-icon>
+      <span class="mx-1">{{ nRecords }}</span>
+      <span translate>record.metadata.publications</span>
```

Comment:
Could be cropped as well to be sure it does not break the design, even though it shouldn't, regarding its expected length.

Reply:
I think it's fine, the translation should not be long, and it's meant to be on the same line